prometheus/src/main/scala/filodb/prometheus/ast/Expressions.scala

```diff
@@ -28,21 +29,41 @@ trait Expressions extends Aggregates with Functions {
       vectorMatch.get.validate(operator, lhs, rhs)
     }
 
-
+    //scalastyle:off
```

Comment:
Why is this needed?  Can we please add specific reason to escape scalastyle here, or just fix/refactor the code?